pkg/sql/copy/copy_in_test.go

```diff
@@ -8,7 +8,7 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
-package sql_test
+package copy_test
```

Comment:
this can just be `package copy` now right

Reply:
Umm, IDK, what's the motivation? Is the idea to use _test only when its necessary to break a package dep cycle?    I thought _test was good hygiene for the only test/use the public bits perspective (which I don't really buy but not the hill I want to die on).   Curious your thoughts...

Reply:
> _test only when its necessary to break a package dep cycle

that was my understanding. that and it has caused me headaches in the past when doing build work :P but not important here.

>  I thought _test was good hygiene for the only test/use the public bits perspective

eh, usually for tests you want to do unit tests with right

----

either way, not a hill i'm looking to die on either.

Reply:
I'm gonna go with if you remove _test and everything still works then it shouldn't be there as a good principle here.  Thanks!